keras/saving/saved_model/load.py

```diff
@@ -1057,10 +1057,15 @@ def keras_api(self):
     return self._serialized_attributes.get(constants.KERAS_ATTR, None)
 
   def get_config(self):
-    if hasattr(self, '_config'):
+    try:
       return self._config
-    else:
-      raise NotImplementedError
+    except AttributeError as e:
+      raise NotImplementedError(
+        "`_config` not found on layer. "
```

Comment:
"`_config` not found on layer." is not a message that end users can understand. We need to provide better context.

Reply:
I agree. That's why I gave the context on the next couple of lines (there would not be a line break once raised). Not sure if you noticed. But I guess you're right in either case and I should rephrase s.t. the higher level info comes first.

In either case, as this PR is probably going to be closed, never mind :-)